src/lib/form/file-uploader.tsx

```
@@ -1,158 +1,145 @@
+  FileTrigger,
+  DropZone,
+  type FileTriggerProps,
+  type FileDropItem,
+  Button,
+} from "react-aria-components";
+import clsx from "clsx";
+
+interface FileUploaderProps {
+  /** Callback function that passes the selected file as argument. */
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
   callback: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)